rpc/answer.go

```diff
@@ -185,14 +185,16 @@ func (ans *answer) setBootstrap(c capnp.Client) error {
 func (ans *answer) Return(e error) {
 	ans.c.lk.Lock()
 	if e != nil {
-		rl := ans.sendException(e)
+		rl := &releaseList{}
+		ans.sendException(rl, e)
 		ans.c.lk.Unlock()
-		rl.release()
+		rl.Release()
 		ans.pcalls.Wait()
 		ans.c.tasks.Done() // added by handleCall
 		return
 	}
-	rl, err := ans.sendReturn()
+	rl := &releaseList{}
+	err := ans.sendReturn(rl)
```

Comment:
Nit:  this could be inlined with the `if` statement below.  It's marginally cleaner that way, IMO.